recipe/meta.yaml

```diff
@@ -1,6 +1,6 @@
 {% set version = "0.5.0" %}  # PEP 386
```

Comment:
looks like non-related to clang build

Reply:
I intended to fix CI with this fix. To commit I need a greed CI